### PR TITLE
Tweaks the Kitchen's Plastic Bag Amount

### PR DESCRIPTION
**Pull request by SleepyGemmy**
title. fixes an oversight with how many plastic bags the kitchen got and increases the plastic bags in the box and vendor from 6 to 20.

### CODE REVIEW

**Comment by SleepyGemmy** [comment]
!review

**Comment by SleepyGemmy** [comment]
!bugfix

**Review by Alberyk — APPROVED** [review]

File: code/game/machinery/vending_types.dm [hunk 1]
```diff
@@ -921,7 +921,7 @@
 		/obj/item/reagent_containers/ladle = 4,
 		/obj/item/storage/toolbox/lunchbox/nt = 6,
 		/obj/item/reagent_containers/glass/rag = 8,
-		/obj/item/evidencebag/plasticbag = 6,
+		/obj/item/evidencebag/plastic_bag = 20,
```
Comment (Arrow768) [inline code/game/machinery/vending_types.dm:924]: Why the path change?
First we have evidencebag and then we have plastic_bag.
Either change evidence_bag as well or restore plasticbag.
But mixing both in the same path is strange.

**Review by Arrow768 — CHANGES_REQUESTED** [review]

File: code/game/machinery/vending_types.dm (hunk 1 above)
Comment (Arrow768) [reply code/game/machinery/vending_types.dm:924]: Also removed the bugfix label as the path change is not a bug fix

**Comment by SleepyGemmy** [comment]
!bugfix

File: code/game/machinery/vending_types.dm (hunk 1 above)
Comment (SleepyGemmy) [reply code/game/machinery/vending_types.dm]: path restored.

**Comment by SleepyGemmy** [comment]
re-adding the bugfix label as the path was restored to original.

**Review by Arrow768 — APPROVED** [review]

**Pull request merged**